docs/ldml/tr35-keyboards.md

```diff
@@ -94,7 +94,7 @@ The LDML specification is divided into the following parts:
   * [Element: variables](#element-variables)
   * [Element: string](#element-string)
   * [Element: set](#element-set)
-  * [Element: unicodeSet](#element-unicodeset)
+  * [Element: uSet](#element-uset)
```

Comment:
Can we go with lower case? It's gonna be kinda ugly for keyboard devs with this mixed case. It's not really two words any more...

```suggestion
  * [Element: uset](#element-uset)
```